src/appMain/main.cc

```diff
@@ -162,6 +162,7 @@ int32_t main(int32_t argc, char** argv) {
 #endif  // LOG4CXX_LOGGER
```

Comment:
it is marked "not ready for review" in the PR summary , is that correct?

Reply:
This means the PR is not ready for the Project Maintainer review.

Reply:
@KhrystynaDubovyk got it, than you